src/base/baseBrushTool.js

```diff
@@ -0,0 +1,367 @@
+import external from './../externalModules.js';
+import EVENTS from './../events.js';
+import baseTool from './../base/baseTool.js';
+// Utils
+import KeyboardController from '../fancy-tools/shared/KeyboardController.js';
+import isToolActive from '../fancy-tools/shared/isToolActive.js';
+import { getNewContext } from '../util/drawing.js';
+import { COLOR_MAP_ID } from '../stateManagement/brushToolColors.js';
+
+export default class extends baseTool {
+  constructor ({
+    name,
+    strategies,
+    defaultStrategy,
+    configuration,
+    supportedInteractionTypes
+  }) {
+    super({
+      name,
+      strategies,
+      defaultStrategy,
+      configuration,
+      supportedInteractionTypes
+    });
+
+    this.isBrushTool = true;
```

Comment:
similar to `isAnnotationTool`, we can probably drop this and use `instanceof`?

Reply:
Good catch, this was pretty copy-paste before I decided on that. Its not used anywhere, so I'll just remove it.